pkg/controller/redirect/redirect.go

```diff
@@ -25,6 +25,14 @@ import (
 	"github.com/google/exposure-notifications-verification-server/pkg/database"
 )
 
+const (
+	// androidOnboardingRedirect and iosOnboardingRedirect are the URLs to
+	// redirect to for onboarding.
+	androidOnboardingRedirect = "market://search?q=exposure%20notifications"
+	iosOnboardingRedirect     = "ens://onboarding"
+	genericOnboardingRedirect = "https://www.google.com/covid19/exposurenotifications/"
```

Comment:
@mikehelmick this wasn't in the spec, but we may want to add it. If we didn't detect ios or android (e.g. someone hit it in their browser), I decided to redirect to the generic EN landing page.

Reply:
I don't think it needs to be in the spec - as that is for device handling